ComputeAdmin/AzureStack.ComputeAdmin.psm1

```diff
@@ -234,155 +245,208 @@ Function Add-AzSVMImage
 
         Add-AzureRMGalleryItem -GalleryItemUri $galleryItemURI
 
         #cleanup
         Remove-Item $GalleryItem
     }
 
-    Remove-AzureStorageContainer –Name $containerName -Force
+    Remove-AzureStorageContainer -Name $containerName -Force
     Remove-AzureRmStorageAccount -ResourceGroupName $resourceGroupName -AccountName $storageAccountName 
     Remove-AzureRmResourceGroup -Name $resourceGroupName -Force
 }
 
+Export-ModuleMember -Function 'Add-AzsVMImage' 
+
+<#
+    .SYNOPSIS
+    Removes an existing VM Image from your Azure Stack.  Does not delete any maketplace items created by Add-AzSVMImage.
+#>
+function Remove-AzsVMImage {
 
-function Remove-AzSVMImage
-{
-    [CmdletBinding(SupportsShouldProcess=$true)]
+    [CmdletBinding(SupportsShouldProcess = $true)]
     Param(
-        [Parameter(Mandatory=$true)]
-        [ValidatePattern(“[a-zA-Z0-9-]{3,}”)]
-        [String] $publisher,
+        [Parameter(Mandatory = $true)]
+        [ValidatePattern("[a-zA-Z0-9-]{3,}")]
+        [String] $Publisher,
        
-        [Parameter(Mandatory=$true)]
-        [ValidatePattern(“[a-zA-Z0-9-]{3,}”)]
-        [String] $offer,
+        [Parameter(Mandatory = $true)]
+        [ValidatePattern("[a-zA-Z0-9-]{3,}")]
+        [String] $Offer,
     
-        [Parameter(Mandatory=$true)]
-        [ValidatePattern(“[a-zA-Z0-9-]{3,}”)]
-        [String] $sku,
+        [Parameter(Mandatory = $true)]
+        [ValidatePattern("[a-zA-Z0-9-]{3,}")]
+        [String] $Sku,
     
-        [Parameter(Mandatory=$true)]
-        [ValidatePattern(“\d+\.\d+\.\d+”)]
-        [String] $version,
-        
-        [String] $location = $null,
+        [Parameter(Mandatory = $true)]
+        [ValidatePattern("\d+\.\d+\.\d+")]
+        [String] $Version,
+
+        [Parameter(Mandatory = $false)]
+        [String] $Location,
 
         [switch] $KeepMarketplaceItem,
 
         [switch] $Force
     )
         
-    $location = Get-AzSLocation -Location $location
+    $location = Get-AzsLocation -Location $location
         
     $VMImageExists = $false
-    if (Get-AzSVMImage -publisher $publisher -offer $offer -sku $sku -version $version -location $location -ErrorAction SilentlyContinue)
-    {
+    if (Get-AzsVMImage -publisher $publisher -offer $offer -sku $sku -version $version -location $location -ErrorAction SilentlyContinue) {
         Write-Verbose "VM Image is present in Azure Stack - continuing to remove" -Verbose
         $VMImageExists = $true
     }
-    else
-    {
-        Write-Verbose -Message ('VM Image with publisher "{0}", offer "{1}", sku "{2}" is not present and will not be removed. Marketplace item may still be removed.' -f $publisher,$offer,$sku) -ErrorAction Stop
+    else {
+        Write-Verbose -Message ('VM Image with publisher "{0}", offer "{1}", sku "{2}" is not present and will not be removed. Marketplace item may still be removed.' -f $publisher, $offer, $sku) -ErrorAction Stop
```

Comment:
There are a bunch of these... I am inclined to leave it as mismatched casing is acceptable in powershell.   Capitalization of parameters though can look bad in functions properties.   Lets leave it for now.